harc_game_web/apps/posts/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Post
```

Comment:
it is preferred to specify full path in import, in this case `apps.posts.models`